src/Codeception/Module/Db.php

```diff
@@ -869,7 +869,7 @@ public function grabColumnFromDatabase($table, $column, array $criteria = [])
      * @param string $column
      * @param array  $criteria
      *
-     * @return array
+     * @return string|bool
```

Comment:
This method can return other types too, so it would be the best to change it to `@return mixed`

Reply:
Ok, at least `null` can be returned. Agree.

Reply:
Done.

Reply:
integer too and probably float, because some database libraries return native types.